_test/dokuwikipage.test.php

```diff
@@ -0,0 +1,34 @@
+<?php
+/**
+ * General tests for the findologicxmlexport plugin
+ *
+ * @group plugin_findologicxmlexport
+ * @group plugins
+ */
+
+require_once(__DIR__ . '/../DokuwikiXMLExport.php');
+require_once(__DIR__ . '/../DokuwikiPage.php');
+require_once(__DIR__ . '/../admin.php');
+require_once(__DIR__ . '/../_test/Helper.php');
+
+class dokuwikipage_test extends DokuWikiTest
+{
+    public function setUp()
+    {
+        Helper::setUp();
+    }
+
+    /**
+     * Test to ensure that a page object contains correct data when saving a page.
+     */
+    public function test_dokuwikipage_creation_returns_a_valid_page() {
```

Comment:
What are you testing here? Isn't this just Dokuwiki stuff completely unrelated to the plugin? Or is it about testing `Helper::savePages`? I feel that testing test helpers can be done implicitely because they are used in tests anyway that will fail if the helper method does not work correctly.

Reply:
nvm, just figured out that 'DokuwikiPage` is part of this plugin :)